pkg/ebpf/derivations.go

```diff
@@ -0,0 +1,72 @@
+package ebpf
```

Comment:
I don't see a reason to split these functions to a new file.
It is short enough, and will be easier to read if we will keep it in the same file

Reply:
#1561 and #1583 shouldn't be derived events. I still didn't have the time to review these PRs yet, but from a quick look, the logic there should be moved to process_events.
Even if we will have more derived events in the future, the deriveEvent function is the only generic function there, and all the other deriveFns are the specific ones. By placing the generic deriveEvent function first in the file, I think this should be easier to read and maintain

Reply:
Done